tools/mo/openvino/tools/mo/subprocess_main.py

```diff
@@ -9,11 +9,11 @@
 
 def check_python_version():
     """
-    Checks python version to be greater or equal than 3.4
+    Checks python version to be greater or equal than 3.7
     :return: exit code (1 - error, None - successful)
     """
-    if sys.version_info < (3, 4):
-        print('Python version should be of version 3.4 or newer')
+    if sys.version_info < (3, 7):
+        print('Python version should be of version 3.7 or newer')
         return 1
 
```

Comment:
please revert these changes

Reply:
can we drop a python version check at all here?
what python version specific do we have in this file?

I suppose supported python versions are reflected by wheel binary packages OpenVINO provides. E.g. we don't provide prebuilt packages for 3.6, which means we don't support this version.

Reply:
It will be more easy just do not touch this code with minimal impact.
If we plan to remove this check at all, let us do it in separate PR.

For now it will much easy to revert these changes by spending 5 sec. Rather that to remove this check and will be stucked by some MO unit-test in several hours in the pre-commit.

Reply:
could you please remove this check in a separate PR if it's not really needed? :) 
we will have less questions in future

Reply:
I will do it. For now we can accept this solution.